src/containers/BooksList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from '../components/Book';
+
+function createTableHeader() {
+  return (
+    <tr>
+      <th>Book ID</th>
+      <th>Title</th>
+      <th>Category</th>
+    </tr>
+  );
+}
+
+function BooksList() {
+  const { books } = useSelector(state => state);
+  return (
+    <table>
+      <thead>
+        { createTableHeader() }
+      </thead>
+      <tbody>
+        { books.map(book => (
+          <Book
+            key={book.bookId}
+            id={book.bookId}
+            title={book.title}
+            category={book.category}
+          />
+        )) }
```

Comment:
- pass the `book` object as a prop to each `Book` component  (pass it as an OBJECT, not split as you did). Do something like :
``` 
 <Book key={index} book={book} />
```